gen/src/generate/ucd/mod.rs

```diff
@@ -39,10 +39,12 @@ pub fn generate() -> io::Result<()> {
     fs::create_dir_all(path)?;
     core::generate(path, &ucd_version, &unicode_data)?;
 
-    let path = Path::new("unic/ucd/normal/src/tables");
-    let _ = fs::remove_dir_all(path);
-    fs::create_dir_all(path)?;
-    normal::generate(path, &ucd_version, &unicode_data)?;
+//    let path = Path::new("unic/ucd/normal/src/tables");
```

Comment:
Would you please make sure the comment markers are added *indented* (not at the beginning of the line)?